interface/therapy_groups/therapy_groups_views/listGroups.php

```diff
@@ -25,6 +25,7 @@
  */
 ?>
 <?php require 'header.php'; ?>
+<?php require_once("{$GLOBALS['srcdir']}/formatting.inc.php"); ?>
```

Comment:
This is not needed since this library is already autoloaded:
https://github.com/matrix-israel/openemr/blob/d65173292746c001e94694c906dd28b0494a01db/composer.json#L45